firmware/can_gateway/src/display.cpp

```diff
@@ -154,22 +154,49 @@ void update_display( void )
 
 static void update_leds( void )
 {
+    static uint32_t lastToggle = 0;
+    static uint8_t toggleState = 0;
+
     if( (g_display_state.status_screen.brakes == MODULE_STATUS_ERROR)
         || (g_display_state.status_screen.steering == MODULE_STATUS_ERROR)
         || (g_display_state.status_screen.throttle == MODULE_STATUS_ERROR) )
     {
         g_display.enableRedLed( );
     }
-    else if( (g_display_state.status_screen.brakes == MODULE_STATUS_UNKNOWN)
-        || (g_display_state.status_screen.steering == MODULE_STATUS_UNKNOWN)
-        || (g_display_state.status_screen.throttle == MODULE_STATUS_UNKNOWN) )
+    else if( (g_display_state.status_screen.brakes == MODULE_STATUS_DISABLED)
+        && (g_display_state.status_screen.steering == MODULE_STATUS_DISABLED)
+        && (g_display_state.status_screen.throttle == MODULE_STATUS_DISABLED) )
     {
-        g_display.enableYellowLed( );
+	if(toggleState)
```

Comment:
The initial value of `toggleState` is set to 0, and I don't see where it'll be set to 1. How will this region of code ever be executed?

Reply:
Scratch that, @rebpdx showed me what's going on. This conditional nesting is pretty confusing, can we collapse this to something like

```c
if(toggleState && millis() - lastToggle > 500)
{
    // enable LED
}
else if(millis() - lastToggle > 1000)
{
    // disable LED
}
```

Reply:
Just two different ways of writing the same thing.  TBH I think yours is confusing :)
I would recommend always explicitly defining order of operations using parens with embedded software.  It helps show other humans your intentions and it eliminates the possibility of an order of ops goof.